mungegithub/mungers/close-stale.go

```diff
@@ -109,6 +99,20 @@ func findLastHumanPullRequestUpdate(obj *github.MungeObject) (*time.Time, bool)
 	return lastHuman, true
 }
 
+func findLastHumanPullRequestUpdate(obj *github.MungeObject) (*time.Time, bool) {
+	pr, ok := obj.GetPR()
+	if !ok {
+		return nil, ok
+	}
+
+	comments, ok := obj.ListReviewComments()
+	if !ok {
+		return nil, ok
+	}
+
+	return findLastHumanPullRequestUpdateHelper(pr.CreatedAt, comments)
```

Comment:
@grodrigues3 Noted! `LastComment()` returns modification time of last `IssueComment`. To determine whether a PR is inactive, i think we need to compare last comment of both `IssueComment` and `PullRequestComment`. Reviewer may post an `IssueComment` with general feedback that is not relevant to any code.

Should i add some helper functions to `/matchers/comment.go` handling the case for `github.PullRequestComment`?